### ISSUE TITLE
Error: $ operator is invalid for atomic vectors

### ISSUE THREAD

**Issue by paulstaab** [statgenlmu/jaatha#76]
Run 93 of 2.99.0.9204/tt_middle failed with

```
Error: $ operator is invalid for atomic vectors 
x$pars_normal
```

**Comment by paulstaab** [comment]
Seems to result from failed calculation of iHS:

```R
  .. ..- attr(*, "condition")=List of 2
  .. .. ..$ message: chr "'x' must be an array of at least two dimensions"
  .. .. ..$ call   : language colSums(haplohh@haplo == 1)
  .. .. ..- attr(*, "class")= chr [1:3] "simpleError" "error" "condition"
```

**Comment by paulstaab** [comment]
Should be fixed with statgenlmu/coala#101
